clippy_utils/src/ty.rs

```diff
@@ -1101,7 +1101,7 @@ pub fn make_projection<'tcx>(
 ///
 /// This function is for associated types which are "known" to be valid with the given
 /// substitutions, and as such, will only return `None` when debug assertions are disabled in order
-/// to prevent ICE's. With debug assertions enabled this will check that that type normalization
+/// to prevent ICE's. With debug assertions enabled this will check that type normalization
```

Comment:
sometimes `that that` is intentional, and occasionally it's necessary, I don't think it's necessary here, and in fact, I suspect that in this case the second `that` wouldn't properly reference a thing that is type normalization.

Reply:
As the person who wrote that, that is a typo. One I do way too frequently.